### ISSUE TITLE
getCurrent url

### ISSUE THREAD

**Issue by SSchaaf** [strongloop/loopback-sdk-angular#63]
getCurrent url is not ok.

https://github.com/strongloop/loopback-sdk-angular/blob/master/lib/services.template#L111

```javascript
url: urlBase + "/" + <%-: meta.getPath() | q %> + "/:id"
```
need to be 

```javascript
url: urlBase + <%-: meta.getPath() | q %> + "/:id"
````

**Comment by mattwoolnough** [comment]
Just check on my install (Version 1.3.7) & seems to be fixed now. 

"getCurrent": {
           url: urlBase + <%-: meta.getPath() | q %> + "/:id",

**Comment by bajtos** [comment]
Yep, it was fixed by #100.
